02_configure_host.sh

```diff
@@ -106,7 +106,7 @@ if [ "$MANAGE_INT_BRIDGE" == "y" ]; then
     if [ "$INT_IF" ]; then
         echo -e "DEVICE=$INT_IF\nTYPE=Ethernet\nONBOOT=yes\nNM_CONTROLLED=no\nBRIDGE=baremetal" | sudo dd of=/etc/sysconfig/network-scripts/ifcfg-$INT_IF
         if sudo nmap --script broadcast-dhcp-discover -e $INT_IF | grep "IP Offered" ; then
-            echo -e "\nBOOTPROTO=dhcp\n" | sudo tee -a /etc/sysconfig/network-scripts/ifcfg-baremetal
+            grep -q "BOOTPROTO=dhcp" /etc/sysconfig/network-scripts/ifcfg-baremetal || (echo -e "\nBOOTPROTO=dhcp\n" | sudo tee -a /etc/sysconfig/network-scripts/ifcfg-baremetal)
```

Comment:
Maybe the problem is that this line assumed `ifcfg-baremetal` but the lines above used `ifcfg-$INT_IF`.  If `$INT_IF` was not `baremetal`, you'd get duplicates in `ifcfg-baremetal` here.

Reply:
I've just submitted a change with the proper ifcfg-$INT_IF. Thanks Russel for finding it!!!